ddtrace/contrib/util.py

```diff
@@ -1,33 +1,16 @@
-from importlib import import_module
-
-
-class require_modules(object):
-    """
-    Context manager to check the availability of required modules.
-    """
-    def __init__(self, modules):
-        self._missing_modules = []
-        for module in modules:
-            try:
-                import_module(module)
-            except ImportError:
-                self._missing_modules.append(module)
-
-    def __enter__(self):
-        return self._missing_modules
-
-    def __exit__(self, exc_type, exc_value, traceback):
-        return False
-
-
-def func_name(f):
-    """
-    Return a human readable version of the function's name.
-    """
-    if hasattr(f, '__module__'):
-        return "%s.%s" % (f.__module__, getattr(f, '__name__', f.__class__.__name__))
-    return getattr(f, '__name__', f.__class__.__name__)
-
-
-def module_name(instance):
-    return instance.__class__.__module__.split('.')[0]
+# [Backward compatibility]: keep importing modules functions
```

Comment:
It's more a matter of style, but I definitely prefer having `ddtrace.utils` with all our utils, rather than 2 different places with bunch of functions without a logical connection.

Reply:
If that's the API we agree on, then you can remove this comment as it makes the reader think that this is a legacy interface only for BC. Instead of understanding that this is the main interface.

Also, should this move to `utils/__init__.py` instead?

Reply:
Maybe I misunderstood your comment. The main interface is `ddtrace.utils` for internal utilities, so all others (`contrib/util.py` and `util.py`) will be removed in 1.0. I would prefer to keep everything inside `ddtrace.utils` rather than splitting it in `ddtrace.utils` and `ddtrace.contrib.utils`.

In Django, for instance, you have everything under this package and then they're separated in multiple modules by functionality.

Reply:
Alright my bad! Got it after a second read :)